src/style.css

```
@@ -0,0 +1,95 @@
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.search-form {
+  position: relative;
+  margin-top: 20px;
+  align-items: center;
+}
+
+.search-form__input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  margin-top: 20px;
+  margin-bottom: 20px ;
+  padding-left: 62px;
+  display: flex;
+  color: #3d4e61;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+.search-form__input:hover {
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
```

```suggestion
.search-form__input:hover {
  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
}
```
looks like need to change just box-shadow. do not set on hover styles which not changed from normal state